assets/jquery/jquery-query-builder/css/query-builder.dark.css

```diff
@@ -0,0 +1,128 @@
+/*!
```

Comment:
Everything in this directory `assets/jquery/jquery-query-builder` is 3rd party code.